src/app/plugins/animal.plugin.ts

```diff
@@ -0,0 +1,81 @@
+import Constants from '../../common/constants';
+import { Plugin } from '../../common/plugin';
+import { ChannelType, IContainer, IMessage, IHttpResponse} from '../../common/types';
+import { RichEmbed } from 'discord.js';
+
+export class AnimalPlugin extends Plugin {
+  public name: string = 'Animal Plugin';
+  public description: string = 'Delivers pictures of our furry friends!';
+  public usage: string = 'animal';
+  public pluginAlias = ['animal', "Animal"];
+  public permission: ChannelType = ChannelType.Public;
+  public pluginChannelName: string = Constants.Channels.Public.Pets;
+
+  private _API_URL: string = 'https://fetchit.dev/breeds/';
+  constructor(public container: IContainer) {
+    super();
+  }
+
+  public async execute(message: IMessage, args?: string[]) {
+    const parsedArgs = (args || []).map((str) => str.toLowerCase()).join(' ');
+    const name = parsedArgs.charAt(0).toUpperCase() + parsedArgs.slice(1);
+    if (parsedArgs == "help"){
+      const url = this._API_URL + "allbreeds/";
+      this.ListName(url, message);
+    }
+    else{
```

Comment:
most people doing the project like to do it like `} else {` not sure if @joey-colon  and @c650  want to follow this rule.  So pester them.  (line 25,26) also in other places not gonna mention them though.

Reply:
should install and use the `prettier` vs code formatter.